### DISCUSSION TITLE
How to package the neuron app?

### DISCUSSION THREAD

**Discussion by seanbei** [emqx/neuron#868] [Q&A]
I followed the cross compiling steps in host Debian and got the final neuron under the build directory, then I would like to install it to my target board. But downloading the neuron app itself didn't work as below warning:
`./neuron: error while loading shared libraries: libneuron-base.so: cannot open shared object file: No such file or directory`
So how to package all the dependent libraries together with neuron app?

**Comment by seanbei** [comment]
I fixed error one by one after run `./neuron` on my target board, below is what's missing:

- config/
- libneuron-base.so
- libzlog.so.1.2
- persistence/
- plugins/

Where to find them:
1. `libzlog.so.1.2` is under the `zlog` cross-compiling directory: `zlog/src/`
2.  The left is all under `neuron/build` directory, i.e. the directory where `neuron` app is in.

**Comment by fengzeroz** [comment]
You can download Neuron Debian package from https://neugates.io/downloads, then unzip it and you can view the directory of the required relevant files.
